fixtures/get-platform-proxy-remote-bindings/tests/index.test.ts

```diff
@@ -223,17 +237,27 @@ if (auth) {
 					"utf8"
 				);
 
-				const { env, dispose } = await getPlatformProxy<{
-					MY_WORKER: Fetcher;
-				}>({
-					configPath: "./.tmp/config-with-invalid-account-id/wrangler.json",
-				});
-
-				const response = await fetchFromWorker(env.MY_WORKER, "OK", 10_000);
-				// The worker does not return a response
-				expect(response).toBe(undefined);
+				await expect(
+					getPlatformProxy<{
+						MY_WORKER: Fetcher;
+					}>({
+						configPath: "./.tmp/config-with-invalid-account-id/wrangler.json",
+					})
+				).rejects.toMatchInlineSnapshot(
+					`[Error: Failed to start the remote proxy session. There is likely additional logging output above.]`
+				);
 
-				await dispose();
+				expect(errorSpy).toHaveBeenCalledOnce();
+				expect(
+					`${errorSpy.mock.calls?.[0]?.[0]}`
+						// Windows gets a different marker for ✘, so let's normalize it here
+						// so that this test can be platform independent
+						.replaceAll("✘", "X")
```

Comment:
nit: it would be good to have a common helper for that - could be addressed in a follow up PR